templates/project/Cargo.toml

```diff
@@ -20,8 +20,15 @@ embedded-hal = { version = "0.2.7", features = ["unproven"]}
 log = "0.4"
 async-channel = "2"
 futures-lite = "1"
-micro-rdk = {version = "0.2.1", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart","provisioning"], rev = "deb0687" }
+micro-rdk = {version = "0.2.1", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart", "provisioning"{% if camera == "true" %}, "camera" {%endif%}], rev = "deb0687" }
```

Comment:
This is getting a little unwieldy as a one-liner. Are we permitted to make it multi-line?:
```
micro-rdk = {
    version = ...,
    git = ...,
    features = [
        "esp32",
        ...
        {% if camera ... }.
    ],
}